### PR TITLE
DOC - Updated EEGLAB documentation

### PR DESCRIPTION
**Pull request by LucaLaFisca**
I have left the "GitHub style images" so that you can review them. I'll remove the corresponding lines if everything is OK for you.

### CODE REVIEW

File: getting_started/eeglab.md [hunk 1]
```diff
@@ -9,34 +9,66 @@ tags: [dataformat, eeglab, eeg]
...
+- (FieldTrip -> EEGLAB) For data that has been processed in FieldTrip, it can be converted to EEGLAB format using **[fieldtrip2eeglab](https://github.com/fieldtrip/fieldtrip/blob/release/external/eeglab/eeglab2fieldtrip.m)** function. Note that the data (second argument) have to be passed as an array (not a cell):
+```
+eeglab_data = fieldtrip2eeglab(ft_data.hdr,cat(3,ft_data.trial{:}));
+pop_saveset(eeglab_data, 'filename', 'myfilename.set')
+```
+
+** THIS EXAMPLE CODE SHOULD BE CHANGED ACCORDING TO A NEW "fieldtrip2eeglab" FUNCTION THAT CONSIDERS FIELDTRIP DATA STRUCTURE **
```
Comment (robertoostenveld) [inline getting_started/eeglab.md:58]: I suggest to leave out the comment here, and make an issue to change fieldtrip2eeglab in the appropriate repository. After the issue is resolved, we can update the documentation here, explaining that one can do either


    eeglab_data1 = fieldtrip2eeglab(hdr, dat, evt); where evt is optional

or 

    eeglab_data2 = fieldtrip2eeglab(data); % where data is a FT raw structure

File: getting_started/eeglab.md [hunk 2]
```diff
@@ -9,34 +9,66 @@ tags: [dataformat, eeglab, eeg]
...
+
+## Design philosophies
+
+Both EEGLAB and FieldTrip work with data structures in MATLAB memory. The design philosophy in EEGLAB is to gather all data from one subject in a single "EEG" structure, and all data from a group of subjects in a "STUDY" structure. This is different from the design philosophy of FieldTrip, which does not gather all results in a single structure, but keeps the results from different analyses in [different structures](/faq/how_are_the_various_data_structures_defined). The following example shows this philosophycal difference:
+
+![FieldTrip-EEGLAB philosophy](/assets/img/getting_started/eeglab/eeglab_FieldTrip_philosophy.png)
+{% include image src="/assets/img/getting_started/eeglab/eeglab_FieldTrip_philosophy.png" width="300" %}
+
+
+Together with the EEGLAB developers we maintain two functions for converting the data back and forth: **fieldtrip2eeglab** and **eeglab2fieldtrip**.
```
Comment (robertoostenveld) [inline getting_started/eeglab.md:74]: can you add links (to github) to the two functions here?

File: getting_started/eeglab.md [hunk 3]
```diff
@@ -9,34 +9,66 @@ tags: [dataformat, eeglab, eeg]
...
 
-Exporting data from EEGLAB to FieldTrip most conveniently done on the MATLAB command line using the **eeglab2fieldtrip** function. That function takes an "EEG" MATLAB structure as input, and will give a MATLAB structure as output that is identical to the output format of one of the standard FieldTrip functions, for example **[ft_preprocessing](https://github.com/fieldtrip/fieldtrip/blob/release/ft_preprocessing.m)**, **[ft_timelockanalysis](https://github.com/fieldtrip/fieldtrip/blob/release/ft_timelockanalysis.m)** or **[ft_freqanalysis](https://github.com/fieldtrip/fieldtrip/blob/release/ft_freqanalysis.m)**.
+![How does EEGLAB use FieldTrip](/assets/img/getting_started/eeglab/EEGLAB_uses_FieldTrip.png)
+{% include image src="/assets/img/getting_started/eeglab/EEGLAB_uses_FieldTrip.png" width="300" %}
 
-The alternative to **eeglab2fieldtrip** is to save the data to disk and use the standard FieldTrip reading and preprocessing functions to read the data directly from disk.
 
-## Converting data between EEGLAB and FieldTrip
+Note: DIPFIT graphical interface can be used in EEGLAB, but FieldTrip should be installed on the MATLAB path.
```
Comment (robertoostenveld) [inline getting_started/eeglab.md:29]: this sentence is still a bit weird. I think it should be removed.

File: getting_started/eeglab.md [hunk 4]
```diff
@@ -9,34 +9,66 @@ tags: [dataformat, eeglab, eeg]
...
 
-## Using other FieldTrip functions
+## How does EEGLAB use FieldTrip?
 
-Besides using FieldTrip as the underlying "engine" for the DIPFIT plug-in of EEGLAB, it is also possible to export various aspects of your EEGLAB data structure completely to FieldTrip. Subsequently, you can continue your analysis of the data using the command-line FieldTrip functions
+EEGLAB integrates FieldTrip functions through the DIPFIT plug-in. This tool allows the localisation of the source signals from the components that have been separated using ICA. The main function used by DIPFIT is **[ft_dipolefitting](https://github.com/fieldtrip/fieldtrip/blob/release/ft_dipolefitting.m)**. It performs a grid search and non-linear fit with one or multiple dipoles and try to find the location where the dipole model is best able to explain the measured EEG topography, using realistic BEM and FEM volume conduction models.
```
Comment (robertoostenveld) [inline getting_started/eeglab.md:22]: EEGLAB integrates FieldTrip functions through the DIPFIT and the FILE-IO plug-ins. 

<new paragraph> 

The DIPFIT plugin allows the localisation ... (including figure)

<new paragraph> 

The FILE-IO plugin allows reading raw data from disk into EEGLAB, from any of the file formats that is supported by FieldTrip. Most EEG file formats are directly supported in EEGLAB, but for some MEG and iEEG file formats this plugin is needed.

Comment (robertoostenveld) [reply getting_started/eeglab.md:22]: _this is how I propose to change it_

**Review by robertoostenveld — CHANGES_REQUESTED** [review]

File: getting_started/eeglab.md (hunk 3 above)
Comment (LucaLaFisca) [reply getting_started/eeglab.md:29]: I decided to just modify it in a clearer way, finally.

File: getting_started/eeglab.md (hunk 1 above)
Comment (robertoostenveld) [reply getting_started/eeglab.md]: for reference, see https://github.com/fieldtrip/fieldtrip/issues/1857

**Comment by robertoostenveld** [comment]
this is now ready to be merged, right?

**Comment by LucaLaFisca** [comment]
Yes it is. We can merge it.

**Comment by LucaLaFisca** [comment]
> Yes it is. We can merge it.

After having removed the GutHub style images

**Comment by robertoostenveld** [comment]
oops. it is already merged. 

Don't worry, I will update those in the master branch.

**Pull request merged**